app/forms/sufia/forms/permission_template_form.rb

```diff
@@ -54,6 +71,45 @@ def grant_admin_set_access(manage_grants)
           admin_set.edit_groups = manage_grants.select { |x| x[:agent_type] == 'group' }.map { |x| x[:agent_id] }
           admin_set.save!
         end
+
+        # Select appropriate radio button under Release "Varies" option based on saved permission_template
+        def select_release_varies_option(permission_template)
+          # Ignore 'no_delay' or 'fixed' values, as they are separate options
+          unless permission_template.release_period == Sufia::PermissionTemplate::RELEASE_TEXT_VALUE_NO_DELAY ||
+                 permission_template.release_period == Sufia::PermissionTemplate::RELEASE_TEXT_VALUE_FIXED
+            # If release_period BEFORE a specified date, then corresponding option under "varies" was selected
+            if permission_template.release_period == Sufia::PermissionTemplate::RELEASE_TEXT_VALUE_BEFORE_DATE
+              @release_varies = Sufia::PermissionTemplate::RELEASE_TEXT_VALUE_BEFORE_DATE
+            elsif !permission_template.release_period.blank?
+              # If release_period is some other value, it is specifying an embargo period (e.g. 6mos, 1yr, etc)
+              @release_varies = 'embargo'
+              @release_embargo = permission_template.release_period
+            end
```

Comment:
Is it possible to hit for both the `if` and `elsif` branches to be `false`? If so, what happens? Should we add an `else` clause that raises an exception?